### ISSUE TITLE
Batch Download

### ISSUE THREAD

**Issue by laura-ibanez** [eskederks/eMAGMA-tutorial#3]
Hi,

I am trying to go through your tutorial and it seems the zip folders Batche1 to 6 are not correctly zipped.
the downloading with wget takes less than a second and when unzipping I get the following error:
`Archive:  Batch1.annot.zip
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of Batch1.annot.zip or
        Batch1.annot.zip.zip, and cannot find Batch1.annot.zip.ZIP, period.
`

Thank you very much for sharing this new tool! 

Laura

**Comment by eskederks** [comment]
Dear Laura,


Thanks for your email, we'll check the files to see what's going on. @Angela, could you please check the files?


Eske


________________________________

**Comment by AngelaMinaVargas** [comment]
Hi Laura, 
Thank you for using the eMAGMA tutorial.
I have checked the files and they are ok. It looks like that error you that are getting comes up when once tries to import the files using the link to the main page. This won’t work for zip files or big files. 
To download the files with wget , once need to copy the link from the **raw page** of the file, not from the main page. 
I have added the link to raw page to the example with Batch1 in Part 1. 
Try this: click on the file you want to download, this will direct you to the raw page of the file, then do right click on download and copy the link, then in your console use wget: [paste the link]. 
Then unzip the file.

All best
A.
